app/templates/main/record-view.html

```diff
@@ -0,0 +1,32 @@
+{% if file_type == "iiif" %}
```

Comment:
when is this file_type not iiif?

Reply:
File type will only be iiif for files that can be rendered by iiif & UV, see:

```
def get_file_details(file):
    """Retrieve file metadata and determine file type and extension."""
    file_metadata = get_file_metadata(file.FileId)
    file_extension = file.FileName.split(".")[-1].lower()

    if file_extension in ["pdf", "png", "jpg", "jpeg"]:
        file_type = "iiif"
    else:
        file_type = None

    return file_metadata, file_type, file_extension
```

If a render isn't available, we default to loading the record details view

Reply:
Just wanted to check that having this conditional here was ok?

Reply:
sorry for the late response, I get it now. I assume @ltrasca expands this in his work to show the we don't support your document for render ticket?